integration-tests/src/chains/paseo/genesis.rs

```diff
@@ -20,8 +21,8 @@ fn session_keys(
 	para_assignment: AssignmentId,
 	authority_discovery: AuthorityDiscoveryId,
 	beefy: BeefyId,
-) -> paseo_runtime::SessionKeys {
-	paseo_runtime::SessionKeys {
+) -> polkadot_runtime::SessionKeys {
```

Comment:
Can we use the same `runtime` alias here please?
```suggestion
) -> runtime::SessionKeys {
```